samza-core/src/main/java/org/apache/samza/zk/ZkKeyBuilder.java

```diff
@@ -96,6 +98,10 @@ public String getJobModelPath(String jobModelVersion) {
   }
 
   public String getJobModelVersionBarrierPrefix() {
-    return String.format("%s/%s/%s/versionBarriers", getRootPath(), JOBMODEL_GENERATION_PATH, JOB_MODEL_UPGRADE_BARRIER);
+    return String.format("%s/%s/%s/versionBarriers", getRootPath(), JOBMODEL_GENERATION_PATH, JOB_MODEL_UPGRADE_BARRIER_PATH);
+  }
+
+  String getTaskLocalityPath() {
```

Comment:
Doesn't need to be public like methods above?

Reply:
No. It can be package private since the zk implementation using it lies within the same package.

Reply:
Minor: Does that apply to all the methods above? If so, I'd prefer to be consistent and update them all.

Reply:
Done.